virttest/utils_v2v.py

```diff
@@ -110,6 +123,54 @@ def get_cmd_options(self, params):
         self.new_name = self.params.get('new_name')
         self.net_vm_opts = ""
 
+        def _get_input_transport_options():
+            """
+            Set input transport options for v2v
+            """
+            options = ''
+            if self.input_transport is None:
```

Comment:
those code doesn't need if you just return options at the last line of this methold

Reply:
We need it to return if user didn't pass any input_transport options.